Workspaces/MiniWorld/Scripts/MiniWorld.cs

```diff
@@ -96,7 +96,7 @@ private void OnEnable()
 
 		private void OnDisable()
 		{
-			if ((referenceTransform.hideFlags & HideFlags.DontSave) != 0)
+			if ((referenceTransform.hideFlags & HideFlags.HideAndDontSave) != 0)
```

Comment:
I traced this back to the original version of the file. Since I can't think of a reason for it, I replaced it with just if (referenceTransform)